FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -117,6 +117,9 @@ - (instancetype)init:(RCNConfigFetch *)configFetch
     _options = options;
     _namespace = namespace;
 
+    /// Set retry seconds to a random number between 1 and 6 seconds.
```

Comment:
Any reason this is 1-6 seconds while below there's a 1-4+1 and 1-3 for another use case?

I wonder if we should just unify here and also create a constant for the threshold number

Reply:
They are all different b/c they're for different mechanisms. One is for when the fetch should occur so that we prevent a DDoS attack and we do it from 0 - 3 seconds. Another is for the base http retry time which we want from 1 - 6 seconds so that all app instances don't retry at the same time. And the last one is 2 - 5 for how much we want to multiply base retry time by, this is for exponential back-off if the last attempt failed.